lib/conjur/debify.rb

```diff
@@ -650,7 +650,7 @@ def detect_component
       if %w(master origin/master).include?(branch)
         'stable'
       else
-        'testing'
+        branch
```

Comment:
Here we need to replace any `/` with `.` in the component names, to make Artifactory happy.

We're already doing this here, but it would be nice if debify just did it for us.

I noted this here: https://github.com/conjurinc/appliance/blob/master/APT.md#apt-distribution-and-component